porcupine/default_keybindings.tcl

```diff
@@ -60,6 +60,9 @@ event add "<<Menubar:Edit/Find and Replace>>" <$control_ish-f>
 # fold plugin
 event add "<<Menubar:Edit/Fold>>" <$alt_ish-f>
 
+# comment/uncomment plugin
+event add "<<Menubar:Edit/Comment//uncomment selected lines>>" <Shift-slash>
```

Comment:
No idea why I suggested Shift-slash. On my keyboard layout, slash means Shift+7 and I wanted this to be Ctrl+Shift+7, so the correct thing is:
```suggestion
event add "<<Menubar:Edit/Comment//uncomment selected lines>>" <$control_ish-slash>
```

Reply:
Thank you!!!!